docs/quickstart-packet.md

```diff
@@ -0,0 +1,269 @@
+# How To Install Kubernetes On Packet Cluster Using KubeOne
+
+In this quick start we're going to show how to get started with KubeOne on
+Packet. We'll cover how to create the needed infrastructure using our
+example Terraform scripts and then install Kubernetes. Finally, we're going to
+show how to destroy the cluster along with the infrastructure.
+
+As a result, you'll get Kubernetes 1.14.1 High-Available (HA) clusters with
+three control plane nodes and one worker node (which can be easily scaled).
+
+### Prerequisites
+
+To follow this quick start, you'll need:
+
+* `kubeone` v0.6.0 or newer installed, which can be done by following the
+  `Installing KubeOne` section of [the
+  README](https://github.com/kubermatic/kubeone/blob/master/README.md),
+* `terraform` installed. The binaries for `terraform` can be found on the
+  [Terraform website](https://www.terraform.io/downloads.html)
+
+## Setting Up Credentials
+
+In order for Terraform to successfully create the infrastructure and for KubeOne
+to install Kubernetes and create worker nodes you need an API Access Token. You
+can refer to [the official
+documentation](https://support.packet.com/kb/articles/api-integrations) for
+guidelines for generating the token.
+
+Once you have the API access token you need to set the `PACKET_AUTH_TOKEN` and
+`PACKET_PROJECT_ID` environment variables:
+
+```bash
+export PACKET_AUTH_TOKEN=<api key>
+export PACKET_PROJECT_ID=<project UUID>
```

Comment:
This environment variable is not recognized by Terraform. We export the one recognized by Terraform as well.

Reply:
I don't get what's this comment about